samples/join-a-team-using-qr-code/nodejs/Readme.md

```diff
@@ -0,0 +1,106 @@
+---
+page_type: sample
+description: This sample demos a feature where you can join a team using QR code containing the team's id.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "21-12-2021 23:35:25"
+---
+
+# Join a team using QR code sample
+
+This sample demos a feature where user can join a team using QR code containing the team's id.
+
+User can generate a new QR code (contains team id information) and then scan the QR code to join the team.
+
+`Currently, Microsoft Teams support for QR or barcode scanner capability is only supported for mobile clients`
+
+- Type a message to get a card to generate the QR code.
+
+ ![Card](Images/CardWithButtons.png)
+
+- Select the team from dropdown list for which you want to generate the QR code and then click on      'Generate QR' button.
+
+ ![QR Code](Images/QRCode.png)
+
+- Scan the generated QR code to join the team.
+
+ ![Join Team](Images/TeamQR.png)
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account (not a guest account)
+-  [NodeJS](https://nodejs.org/en/)
+-  [ngrok](https://ngrok.com/) or equivalent tunneling solution
+-  [M365 developer account](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) or access to a Teams account with the 
+   appropriate permissions to install an app.
+
+## To try this sample
+
+> Note these instructions are for running the sample on your local machine, the tunnelling solution is required because
+> the Teams service needs to call into the bot.
+
+### 1. Setup for Bot SSO
+a) In Azure portal, create a [Bot Framework registration resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
+
+**Note:** Once the Azure Bot is created, go to navigation pane and under Settings section select Channels and enable Microsoft Teams Channel.
+
+![Configure Channel](Images/BotChannel.png)
+
+b) Add the following permission in app registration
+
+![Install App](Images/Permission.png)
```

Comment:
Rather than proving a screenshots for permissions, it's better to add the permissions as text so that user can directly copy/paste and add the permissions?

Reply:
Previous samples we followed the format of adding image for permissions so added image.